.github/workflows/production.yml

```diff
@@ -96,3 +96,11 @@ jobs:
             }
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.ACTION_MONITORING_SLACK }}
+      # Trigger staging workflow if this workflow was triggered by push to master
+      - name: Checking workflow method and triggering next workflow if needed
+        if: ${{ success() && github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        run: |
+          curl -X POST \
```

Comment:
For doing repository dispatches you can use this action: https://github.com/peter-evans/repository-dispatch

I use it in other actions.